.gitignore

```diff
@@ -7,6 +7,8 @@ manual/source/.introduction.rst.swp
 manual/source/nexus.pdf
 manual/source/nxdl_desc.rst
 manual/source/*.table
+manual/source/classes/*/*/
```

Comment:
Perhaps it will be easier to allow the limited files with negated patterns. I.e.
```
manual/source/**
!manual/source/**/Makefile
!manual/source/index.rst
```